jetson/nav/gate_search/gateStateMachine.cpp

```diff
@@ -160,12 +160,13 @@ std::shared_ptr<GateStateMachine> GateFactory(const std::weak_ptr<StateMachine>&
 void GateStateMachine::publishGatePath() {
     // Construct vector from deque
     std::vector<Odometry> arr(mPath.begin(), mPath.end());
-    SearchPoints gatePathPoints{
-        .search_pattern_size  = (int32_t)arr.size(),
-        .points = arr
+    ProjectedPoints gatePathPoints{
+        .pattern_size  = static_cast<int32_t>(arr.size()),
+        .points = arr,
+        .path_type = "gate"
```

Comment:
renamed from type because type is a keyword for LCM to represent the struct name